mkdocs/docs/api.md

```diff
@@ -1268,6 +1268,24 @@ with table.update_spec() as update:
     update.rename_field("bucketed_id", "sharded_id")
 ```
 
+## Sort order updates
+
+Users can update the sort order on existing tables for new data. See [sorting](https://iceberg.apache.org/spec/#sorting) for more details.
+
+The API to use when updating a sort order is the `update_sort_order` API on the table.
+
+Sort orders can only be updated by adding a new sort order. They cannot be deleted or modified.
+
```

Comment:
I wonder if the name of the method should be set_sort_order then, as update_sort_order possibly implies that it could be a modification?

Reply:
i think either is fine. theres a updateSortOrderSchema function in java https://grep.app/search?f.repo=apache%2Ficeberg&q=updatesort